test/e2e/autoscale_test.go

```diff
@@ -182,7 +238,9 @@ func setup(t *testing.T, class string, metric string, fopts ...rtesting.ServiceO
 		resources:         resources,
 		deploymentName:    resourcenames.Deployment(resources.Revision),
 		domain:            domain,
-		targetUtilization: cfg.ContainerConcurrencyTargetFraction,
+		targetUtilization: cfg.ContainerConcurrencyTargetFraction, // May needs to changed if annotation is used.
+		targetValue:       containerConcurrency,                   // Use concurrency by default. Other metric should change this value.
```

Comment:
Should we enhance `setup` to provide these values and set the annotations accordingly? I feel like that would feel less awkward.

Reply:
Done.